.github/gradle/gradle.gradle

```diff
@@ -23,21 +22,27 @@ static Map<String, String> removeCommonPrefix(List<String> list) {
     Collections.sort(list, { String a, String b -> a.length() - b.length() })
     def max = list[0].length()
     def prefix = ""
+
+    comparisonLoop:
     for(int index = 0; index < max; index++) {
         def currentChar = list[0][index]
         for (String item : list) {
             if (item[index] != currentChar) {
-                for (final def entry in list) {
-                    result.put(entry, entry.substring(prefix.length()))
-                }
-                return result
+                break comparisonLoop
```

Comment:
```
Script '/home/runner/work/NeoGradle/NeoGradle/.github/gradle/gradle.gradle' line: 27

* What went wrong:
Could not compile script '/home/runner/work/NeoGradle/NeoGradle/.github/gradle/gradle.gradle'.
> startup failed:
  script '/home/runner/work/NeoGradle/NeoGradle/.github/gradle/gradle.gradle': 27: Statement labels may not be used in build scripts.
  In case you tried to configure a property named 'comparisonLoop', replace ':' with '=' or ' ', otherwise it will not have the desired effect.
   @ line 27, column 5.
         for(int index = 0; index < max; index++) {
         ^
  
  1 error
```
Quite disappointing. Oh well, revert my suggestions for this.